schemas/edm.xsd

```diff
@@ -764,8 +766,8 @@
         </xs:restriction>
       </xs:simpleType>
       <xs:simpleType>
-        <xs:restriction base="xs:token">
-          <xs:pattern value="Edm\..+" />
+        <xs:restriction base="xs:string">
+          <xs:pattern value="Edm\.[\p{L}\p{Nl}_][\p{L}\p{Nl}\p{Nd}\p{Mn}\p{Mc}\p{Pc}\p{Cf}]{0,}" />

```

Comment:
Is this meant for future primitive Edm types? If so, shouldn't Collections of them also be supported?